basin3d/plugins/usgs.py

```diff
@@ -618,9 +648,8 @@ def list(self, query: QueryMeasurementTimeseriesTVP):
             if feature_id in feature_obj_dict.keys():
                 monitoring_feature = _load_point_obj(
                     datasource=self,
-                    json_obj=feature_obj_dict[feature_id], feature_observed_properties=dict(),
-                    synthesis_messages=synthesis_messages,
-                    observed_property_variables="Find observed property variables at monitoring feature url")
+                    json_obj=feature_obj_dict[feature_id], observed_property_variables=observed_properties,
```

Comment:
```suggestion
                    json_obj=feature_obj_dict[feature_id], 
                    observed_property_variables=observed_properties,
```

Reply:
When building the model objects, I find it easier to read with one argument per line.